PrismFormsGallery/Source/Modules/CommandingModule/CommandingModule.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+ <ItemGroup>
+    <PackageReference Include="Prism.Forms" Version="7.2.0.1347-pre" />
+    <PackageReference Include="Xamarin.Forms" Version="4.1.0.555618" />
+    <PackageReference Include="Refractored.MvvmHelpers" Version="1.4.1-beta" />
+    <PackageReference Include="Prism.DryIoc.Forms" Version="7.2.0.1347-pre" />
```

Comment:
Since this is for a Module go ahead and remove this reference. You shouldn't need it.

Reply:
Done